src/test/java/fi/metatavu/mecm/reader/vcard/ConverterTest.java

```diff
@@ -161,8 +161,26 @@ public void testStatus() throws IOException, JsonParseException, JsonMappingExce
     assertEquals(OffsetDateTime.of(2017, 6, 5, 0, 0, 0, 0, ZoneOffset.ofHours(2)).toInstant(), status.getDeparture().toInstant());
     assertEquals(OffsetDateTime.of(2017, 6, 6, 0, 0, 0, 0, ZoneOffset.ofHours(2)).toInstant(), status.getArrival().toInstant());
   }
+  
+  @Test
+  public void testPrivateNumber() throws IOException, JsonParseException, JsonMappingException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)